composer.json

```diff
@@ -6,30 +6,37 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.3",
+        "php": ">=7.4",
         "ext-json": "*",
-        "drupal/core": "^8.9 || ^9"
+        "drupal/core": "^9.2",
+        "drupal/ctools": "^3.7",
+        "drupal/file_link": "2.0.4",
+        "drupal/pathauto": "^1.8",
```

Comment:
After setting oe_multilingual minimal version to 1.10 didn't see the need to set pathauto minimum version.